src/test/scala/org/scalameter/MethodInvocationCountTest.scala

```diff
@@ -0,0 +1,7 @@
+package org.scalameter
+
+class MethodInvocationCountTest extends InvocationCountMeasurerTest {
+  test("MethodInvocationCountTest.allocations should be deterministic") {
+    checkInvocationCountMeasurerTest(new org.scalameter.examples.MethodInvocationCountTest1)
```

Comment:
Could you invoke this twice on the same object during the test (particularly if SBT proves problematic about re-running the test, as I suggested in the earlier comment).